src/components/DailyChecklist.tsx

```diff
@@ -19,25 +28,49 @@ const Checkbox = ({ task }: { task: string }) => {
         checked={isChecked}
         onChange={handleOnChange}
       />
-      <label htmlFor={task} className="task_text">
-        {task}
-      </label>
+      {toggle ? (
+        <label
+          htmlFor={task}
+          className="task_text"
+          onDoubleClick={() => setToggle(false)}
+        >
+          {val}
+        </label>
+      ) : (
+        <input
+          className="task_text"
+          type="text"
+          value={val}
+          onChange={(event) => {
+            const content = event.target.value;
+            noteService.editNote(day, index, content);
+            setVal(content);
+          }}
```

Comment:
Style this input field if you can to be smaller and not the same as the field to add tasks

Just need to add a unique id or class to the editing input and creation input

Reply:
fdd48ba